vagrant/provisioning/roles/foia-analytical-reports/tasks/main.yml

```diff
@@ -12,7 +12,15 @@
   file:
     path: "{{ root_folder }}/install/pentaho/foia-reports-dw{{ foia_analytical_reports_version_formatted }}"
     state: directory  
+  when: arkcase_version != "" and arkcase_version is version('2023.02', '<')
 
+- name: create install folder
+  become: yes
+  become_user: pentaho
+  file:
+    path: "{{ root_folder }}/install/pentaho/foia-reports-dw-2023.02.01-RC.01"
```

Comment:
Not hardcoding please :). This 'foia-reports-dw-2023.02.01-RC.01' can be taken from a lot of facts

Reply:
Got it!